operator/pkg/reconciliation/reconcile_racks_helpers.go

```diff
@@ -0,0 +1,90 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+package reconciliation
+
+import (
+	"fmt"
+
+	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	"github.com/datastax/cass-operator/operator/pkg/httphelper"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func mapContains(base map[string]string, submap map[string]string) bool {
+	for k, v := range submap {
+		if val, ok := base[k]; !ok || val != v {
+			return false
+		}
+	}
+	return true
+}
+
+func FilterPodListByLabels(pods []*corev1.Pod, labelMap map[string]string) []*corev1.Pod {
+	filtered := []*corev1.Pod{}
+	for _, p := range pods {
+		if mapContains(p.Labels, labelMap) {
+			filtered = append(filtered, p)
+		}
+	}
+	return filtered
+}
+
+func FilterPodListByLabel(pods []*corev1.Pod, labelName string, labelVal string) []*corev1.Pod {
+	filtered := []*corev1.Pod{}
```

Comment:
why not just forward this to `FilterPodListByLabels`?

Reply:
good point